src/components/data/LocalVariablesContent.js

```diff
@@ -17,6 +17,24 @@ import { withErrorReporting } from 'src/libs/error'
 import * as StateHistory from 'src/libs/state-history'
 import * as Utils from 'src/libs/utils'
 
+// Exported for testing
+export const getDisplayedAttribute = arr => {
+  const obj = Object.assign({}, ...arr)
+  return [obj.key, obj.value, obj.description ? obj.description : '']
+}
```

Comment:
We try to
- not mutate `const`s whenever possible
- use [Lodash FP](https://lodash.com/docs/4.17.15) methods whenever possible since they tend to handle corner cases better than native JS counterparts, and have desirable features like being auto-curried
- explicitly pull out only what's necessary

```suggestion
export const getDisplayedAttribute = arr => {
  const { key, value, description } = _.assign(...arr, {})
  return [key, value, description ?? '']
}
```

I like `description ?? ''` because it's nicely concise but if you feel it's too cryptique, I'm fine with the original (`description ? description : ''`)

Reply:
Didn't realize destructuring could work here - neat.  Also, the `??` operator definitely makes things more readable.